src/utils/mailer.js

```diff
@@ -0,0 +1,24 @@
+const nodemailer = require('nodemailer');
+
+const transporter = nodemailer.createTransport({
+  host: 'smtp.gmail.com',
+  port: 587,
+  secure: false,
+  auth: {
+    user: process.env.GOOGLE_ACCOUNT,
+    pass: process.env.GOOGLE_PASSWORD,
+  },
+});
+
+const sendMail = async (targetMail, subject, html) => {
+  const info = await transporter.sendMail({
+    from: process.env.GOOGLE_ACCOUNT,
```

Comment:
I think Nodemailer allows you to format the sender address as : `'"Display Name" <foo@example.com>'`
So you can leverage that feature and add your display name as something like "Team Voluntera".
Then your automated emails will look more professional.

(Extra tip - Add a cool profile picture like a company logo on the email ID if you haven't already. Then receivers will see a sender display name as well as display picture.)